### PR TITLE
Support multiple types in the endpoint listing API

### PR DESCRIPTION
**Pull request by josejulio**
- Add missing properties in camel update

~~edit: need to update tests~~

### CODE REVIEW

**Comment by bennetelli** [comment]
somwhow I can't checkout the branch, but we should fix the failed tests first.

**Comment by josejulio** [comment]
@bennetelli fixed the issue and added a new test

File: backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java [hunk 1]
```diff
@@ -463,6 +463,136 @@ void testEndpointUpdates() {
...
+        camelProperties.setSubType("demo");
+
+        Endpoint camelEp = new Endpoint();
+        camelEp.setType(EndpointType.CAMEL);
+        camelEp.setName("endpoint to find");
+        camelEp.setDescription("needle in the haystack");
+        camelEp.setEnabled(true);
+        camelEp.setProperties(camelProperties);
+
+        response = given()
```
Comment (bennetelli) [inline backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java:519]: I would prefer two tests so its clear what the responsibility of the test is.
We can probably split it in a camel and webhook test don't we?

Comment (josejulio) [reply backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java:519]: The purpose of the test is to actually verify that the `type` param is used correctly and that it supports both 1 type or multiple types.

I could probably make the test more focused If you consider it is necesary

Comment (josejulio) [reply backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java:533]: Parametrized the test, let me know what you think.

Comment (bennetelli) [reply backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointServiceTest.java:533]: That's much better :) thanks

**Review by bennetelli — APPROVED** [review]

**Pull request merged**